package.json

```
@@ -1,75 +1,69 @@
 {
   "name": "@telamonian/theme-darcula",
-  "version": "2.0.0",
-  "description": "Darcula theme for Jupyterlab. Modelled after the classic Intellij theme.",
-  "author": "Max Klein",
+  "version": "3.0.0",
+  "description": "Darcula theme for JupyterLab. Modelled after the classic Intellij theme.",
   "keywords": [
     "jupyter",
     "jupyterlab",
-    "jupyterlab-extension",
-    "jupyterlab-theme",
-    "darcula",
-    "darcula-theme"
+    "jupyterlab-extension"
   ],
+  "homepage": "https://github.com/telamonian/theme-darcula",
+  "bugs": {
+    "url": "https://github.com/telamonian/theme-darcula/issues"
+  },
   "license": "BSD-3-Clause",
+  "author": "Max Klein",
   "files": [
-    "lib/*.d.ts",
-    "lib/*.js.map",
-    "lib/*.js",
-    "style/codemirror-darcula.css",
-    "style/index.css",
-    "style/variables.css"
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
   ],
-  "sideEffects": true,
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "style": "style/index.css",
-  "directories": {
-    "lib": "lib/"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/telamonian/theme-darcula.git"
   },
   "scripts": {
-    "build": "tsc -b",
-    "build:dev": "jlpm install && jlpm build && jupyter labextension install .",
-    "clean": "rimraf lib",
-    "clean:slate": "jlpm run clean && rimraf node_modules",
-    "precommit": "lint-staged",
-    "prepublishOnly": "npm run build",
-    "prettier": "prettier --write '**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}'",
-    "watch": "tsc -b --watch"
-  },
-  "lint-staged": {
-    "**/*{.ts,.tsx,.css,.json,.md}": [
-      "prettier --write",
-      "git add"
-    ]
+    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
+    "build:prod": "jlpm run build:lib && jlpm run build:labextension",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc",
+    "clean": "jlpm run clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf theme-darcula/labextension",
+    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+    "eslint": "eslint . --ext .ts,.tsx --fix",
+    "eslint:check": "eslint . --ext .ts,.tsx",
+    "install:extension": "jupyter labextension develop --overwrite .",
+    "prepare": "jlpm run clean && jlpm run build:prod",
+    "watch": "run-p watch:src watch:labextension",
+    "watch:src": "tsc -w",
+    "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
-    "@jupyterlab/application": "^2.0.0",
-    "@jupyterlab/apputils": "^2.0.0"
+    "@jupyterlab/application": "^3.0.0-rc.13",
+    "@jupyterlab/apputils": "^3.0.0-rc.13"
```

Not tested yet, but this might be enough for JupyterLab 2.x support:

```suggestion
    "@jupyterlab/application": "^2.0.0 || ^3.0.0-rc.13",
    "@jupyterlab/apputils": "^2.0.0 || ^3.0.0-rc.13"
```